R/fill_na.R

```diff
@@ -0,0 +1,19 @@
+#' Replace daily inc with NA in specific rows
+#'
+#' @param results a result data.frame
+#' @param adjustments a adjustment case data.frame
+#' @return a data.frame with MAs on adjustment cases
+#' @export
+fill_na <- function(results, adjustments) {
+  adjustments$dates <- as.character(adjustments$dates)
+
+  results %>%
+    dplyr::rowwise() %>%
+    dplyr::mutate(
+      # for each row, if date is in adjustment date and location is in adjustment states
+      inc = ifelse(date %in% adjustments$dates[adjustments$fips == location] |
```

Comment:
I would have expected the condition to check here to be `&` rather than `|`.  Am I missing something?

If this is a bug, we should be sure we have a unit test that can detect it.